.changeset/bright-dragons-enjoy.md

```diff
@@ -0,0 +1,5 @@
+---
+"@effect/docgen": patch
```

Comment:
```suggestion
"@effect/docgen": minor
```

Reply:
Addressed in https://github.com/Effect-TS/docgen/pull/90/commits/dec3a4303fd35fc5659ed868296b28107a61adf9